pyccel/stdlib/ndarrays/ndarrays.h

```diff
@@ -140,4 +141,16 @@ int64_t         get_index(t_ndarray arr, ...);
 int64_t     *numpy_to_ndarray_strides(int64_t *np_strides, int type_size, int nd);
 int64_t     *numpy_to_ndarray_shape(int64_t *np_shape, int nd);
 
+/* numpy.sign for float, double and integers */
+inline double  sign(double x)
+{
+    return x ? (x < 0 ? -1 : 1) : 0;
+}
+
+/* numpy.sign for complex */
+inline double complex csign(double complex x)
+{
+    return x ? ((!creal(x) && cimag(x) < 0) || (creal(x) < 0) ? -1 : 1) : 0;
+}
```

Comment:
Do we handle the situation where the user has defined a variable named `sign` or `csign`?

CC: @EmilyBourne

Reply:
This case is not handled. To fix it you should add these keywords to the list [here](https://github.com/pyccel/pyccel/blob/master/pyccel/naming/cnameclashchecker.py#L16)

Reply:
@aihya: Please implement Emily's suggestion

Reply:
fixed here https://github.com/pyccel/pyccel/pull/1169/commits/0b718a47c15eb8e5595fe4ef3fb9dbcf27928a83